docker/root/usr/bin/setup

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 
+#TODO maybe /data/git/.ssh not needed anymore
```

Comment:
If docker user internal SSH server, it's not needed

Reply:
While it is not needed for the single process setup, removing it would break existing setups.

Reply:
@das7pad yes but it is for the setup part so if we don't need it anymore we don't need to create it if it doesn't exist.

Reply:
For folks who backup the entire /data/git directory it is not a problem. But for those who just backup non-generated files, read only the repositories directory, the .ssh directory would be missing upon restore.

Reply:
But the internal sshserver doesn't use the .ssh directory...

Reply:
I was thinking about the third example in https://github.com/go-gitea/gitea/pull/7129#pullrequestreview-250188735.
I know that the files in the .ssh directory are only needed to get the spawned sshd-childs talk with the correct gitea instance.